src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -583,6 +657,52 @@ protected ClassLoader configureClassLoader(ClassLoader loader) {
         return context.getServletContext();
     }
 
+    /**
```

Comment:
Not new code, so please only review the _diff_ of this method with the `createWebServer2` method (left as an exercise for the reader).